db/Models/app.models.js

```diff
@@ -46,4 +46,18 @@ function fetchReview(review_id) {
   });
 }
 
-module.exports = { fetchCategories, fetchReviews, fetchReview };
+function fetchComments(review_id) {
+  let queryStr = "SELECT * FROM comments";
+  const queryParams = [];
+
+  if (review_id) {
```

Comment:
Would suggest this endpoint always has a review_id so would not reccomend this complex query like use of building up the query string.